src/hooks/use-model-toggle.ts

```diff
@@ -4,11 +4,10 @@ import {
     onMounted,
     getCurrentInstance,
     Ref,
-    PropType,
-    isVue2
+    PropType
 } from 'vue-demi'
 
-const modelValueNaming = isVue2 ? 'model-value' : 'modelValue'
```

Comment:
thanks to @prodan-eric for checking this behavior we might need to keep this one in as specific components do not work if we dont have this check. 
we will handle this in further components checks so we will need the default check for the default elements.